### PR TITLE
Move next-mdx from zeit/next-plugins to zeit/next.js

### PR DESCRIPTION
**Pull request by ijjk**
At request of @timneutkens I moved the `next-mdx` plugin from the next-plugins repo into Next.js

Depends on https://github.com/zeit/next.js/pull/6443

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 ## Official Next.js plugins
 
-- [@zeit/next-mdx](./packages/next-mdx)
```
Comment (timneutkens) [inline readme.md]: Let's link this to the new location 🕵️

Comment (ijjk) [reply readme.md]: Updated

File: packages/next-mdx/readme.md [hunk 2]
```diff
@@ -1,78 +0,0 @@
-# Next.js + MDX
```
Comment (timneutkens) [inline packages/next-mdx/readme.md]: We should keep the file here but link to the new plugin. Also link to the old docs (a previous commit) if the user wants to read about the old usage.

Comment (ijjk) [reply packages/next-mdx/readme.md]: Updated

Comment (timneutkens) [reply packages/next-mdx/readme.md]: 👌

**Pull request closed without merging**